File: app/src/main/java/com/example/asonbom/data/responses/LoginResponse.java
package com.example.asonbom.data.responses;

import com.google.gson.annotations.SerializedName;

public class LoginResponse {
    @SerializedName("token")
    private String token;

    @SerializedName("user")
    private Object user;


    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }

    public Object getUser() {
        return user;
    }

    public void setUser(Object user) {
        this.user = user;
    }
}

File: app/src/main/java/com/example/asonbom/data/responses/LoginResponse.kt
package com.example.asonbom.data.responses

import com.example.asonbom.data.models.User
import com.google.gson.annotations.SerializedName
import org.json.JSONObject

data class LoginResponse(
    @SerializedName("status_code")
    var statusCode:Int,

    @SerializedName("auth_token")
    var authToken:String,

    @SerializedName("user")
    var user: User?,

    @SerializedName("token")
    var getToken: String,

    @SerializedName("sample_json")
    val sampleJson: JSONObject = JSONObject("""{"email":"moises@moises.com", "password":"admin"}""")
)